boring/src/ssl/mod.rs

```diff
@@ -3469,6 +3450,7 @@ where
         if ret > 0 {
             Ok(stream)
         } else {
+            println!("deez nuts error");
```

Comment:
I think we don't want that in production 😄

Reply:
lmao whoops